### DISCUSSION TITLE
Career in Blockchain with project management

### DISCUSSION THREAD

**Discussion by dornubarii** [Cyfrin/foundry-full-course-cu#1465] [Q&A]
hello, I'm really excited about this course and I'm learning alot but I'm not a developer , i am a project manager and i need guidance on how i fit in

**Comment by EngrPips** [comment]
Hello @dornubarii , There is not really much on fitting in. I would advice you take the course with the pace that is best for you and as far as you are understanding everything that is being taught in the course you are fitting in already. Also make sure to always leverage discussion channel of this github to always ask questions and reach out whenever you are stuck. I wish you the best on your journey of knowledge

**Reply by dornubarii** [reply]
Thank you verry much for this
